Specs/AppLovin/5.0.1/AppLovin.podspec.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "AppLovin",
+  "authors": "AppLovin Corporation",
+  "version": "5.0.1",
+  "platforms": {
+    "ios": "6.0"
```

Comment:
is that actually true?! I believe it's iOS 9 or so

Reply:
I shamelessly copied their official podspec, so whatever is there should be right.